src/black/__init__.py

```diff
@@ -1241,10 +1241,7 @@ def _format_str_once(
     elt = EmptyLineTracker(mode=mode)
     split_line_features = {
         feature
-        for feature in {
-            Feature.TRAILING_COMMA_IN_CALL,
-            Feature.TRAILING_COMMA_IN_DEF,
-        }
+        for feature in {Feature.TRAILING_COMMA_IN_CALL, Feature.TRAILING_COMMA_IN_DEF}
```

Comment:
Was this a result from the change in the PR? I would expect this code to remain unchanged because of the magic trailing comma.